notifier.rst

```diff
@@ -912,3 +912,32 @@ is dispatched. Listeners receive a
 ..    - Describe notification_on_failed_messages integration
 
 .. _`RFC 3986`: https://www.ietf.org/rfc/rfc3986.txt
+.. _`AllMySms`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/AllMySms/README.md
```

Comment:
If this PR gets accepted we can ensure this by a DOCtor rule, that in 5.4 only links to 5.4 are allowed and so on.

WDYT?

Reply:
I prefer link generation if possible instead.
Else at each new release we have to manually update links